oracle/blockeligibilityvalidator.go

```diff
@@ -55,17 +85,24 @@ func (v BlockEligibilityValidator) BlockEligible(block *types.BlockHeader) (bool
 	epochBeacon := v.beaconProvider.GetBeacon(epochNumber)
 	message := serializeVRFMessage(epochBeacon, epochNumber, counter)
 	vrfSig := block.EligibilityProof.Sig
-	res, err := v.validateVRF(message, vrfSig, []byte(block.MinerID.VRFPublicKey))
+
+	nodeId, err := v.activationDb.GetIdentity(pubString)
```

Comment:
I am confused about why this is here.
We already know the block's creator identity from IsIdentityActive().
Seems like you don't use it.

Reply:
me to :)

Reply:
me to...